sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -236,15 +239,36 @@ def construct_test_environment(
     online_store = test_repo_config.online_store
 
     with tempfile.TemporaryDirectory() as repo_dir_name:
+        if test_repo_config.python_feature_server:
+            from feast.infra.feature_servers.aws_lambda.config import (
+                AwsLambdaFeatureServerConfig,
+            )
+
+            feature_server = AwsLambdaFeatureServerConfig(
+                enabled=True,
+                execution_role_name="arn:aws:iam::402087665549:role/lambda_execution_role",
```

Comment:
This seems like it might break users who use Feast for testing their plugins?

Reply:
I think it's probably fine for plugin tests to skip tests against serverless